### ISSUE TITLE
Show not ready to promote WIs in pipeline

### ISSUE THREAD

**Issue by oleg-mastriukov** [forcedotcom/devops-center-roadmap#122]
Currently, to get the list of not ready to promote WIs, we have to go to the WIs list and filter them there making it easy to lose some changes. E.g. when a pull request is waiting for approval on GitHub
I'd like to have the option to show not only ready-to-promote WIs in the pipeline but also an additional list of all WIs that got stuck before they were marked with this checkbox.

**Comment by john-hutchinson** [comment]
Yes please! A kanban view that doesn't show the 'waiting for review' or 'in progress' items is an odd design choice. As a result as @oleg-mastriukov mentions we manage WI's in at least 2 places currently (4 actually due to other limitations but that's an aside. Hoping we can reduce this to 3.)

**Comment by basildobek** [comment]
Current interface is very clunky.  This would help a lot with a common task.
